oaknational/Oak-Web-Application
# Feat/ctc 38/update programme query

## Description

Music year: 1990

### ACs

- [x]  new mv is used in query: published_mv_synthetic_unitvariant_lessons_by_keystage_13_0_0
- [x]  schema is updated
- [x]  overrides and exceptions are implemented
- [x]  navigation is uneffected by changes

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-3002--oak-web-application.netlify.thenational.academy/teachers/key-stages/ks4/subjects/maths/programmes
2. Click on other programme pages
3. You should see no chnges to 

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
